tests/test_shrink.toml

```diff
@@ -0,0 +1,109 @@
+# simple shrink
+[cases.test_shrink_simple]
+defines.BLOCK_COUNT = [10, 15, 20]
+defines.AFTER_BLOCK_COUNT = [5, 10, 15, 19]
+   
+if = "AFTER_BLOCK_COUNT <= BLOCK_COUNT"
+code = '''
+#ifdef LFS_SHRINKIFCHEAP
```

Comment:
Ah, yes. This is the best way to `ifdef` right now. I just noticed the earlier test failures.

I have a branch that adds `ifdef` attrs to the test case itself, but it hasn't been upstreamed yet unfortunately...